ivy/functional/ivy/layers.py

```diff
@@ -1217,12 +1217,12 @@ def depthwise_conv2d(
 def conv3d(
     x: Union[ivy.Array, ivy.NativeArray],
     filters: Union[ivy.Array, ivy.NativeArray],
-    strides: int,
+    strides: Union[int, Tuple[int, int, int]],
```

Comment:
I am not totally sure, but can this be: `strides: Union[int, Tuple[int], Tuple[int, int], Tuple[int, int, int]]`? I noticed similar pattern in the Conv2d implementation.

Reply:
updated now

Reply:
Sorry for back and forth, but I checked the TF and Torch documentation again and it seems like the initial idea was correct. Can I please ask you to try and run a couple of tests and see whether the function can accept these types of inputs. Thanks!

Reply:
it gives "tuple index out of range " as tuples dim3 is accessed in the backend functions, i edited it back as in the docs .idk if that pattern was desired in ivy , the backend funcs should be modified